.github/workflows/release.yml

```diff
@@ -746,9 +746,33 @@ jobs:
       with:
         arch: ${{ matrix.host }}
 
+    - name: Save code signing certificate to disk
+      id: save_cert
+      uses: timheuer/base64-to-file@v1
+      with:
+        filename: 'openttd.p12'
+        # The certificates in a PKCS12 file encoded as a base64 string
+        encodedString: ${{ secrets.WIN_CERTIFICATE_P12 }}
+      # If this is run on a fork, there may not be a certificate set up - continue in this case
+      continue-on-error: true
+
+    - name: Import code signing certificate
+      shell: powershell
+      # If this is run on a fork, there may not be a certificate set up - continue in this case
+      continue-on-error: true
+      env:
+        WIN_CODESIGN_P12: ${{ steps.save_cert.outputs.filePath }}
```

Comment:
`CODESIGN` vs `CERTIFICATE`. Any reason for the naming like this?

Reply:
Not particularly; this bit was perhaps written before I normalised the secret names against the Apple equivalents!